rules.py

```diff
@@ -848,7 +886,8 @@ def check(self):
     ReusedCandidate,
     CoalitionParties,
     ProperBallotSelection,
-    CandidateNotReferenced
+    CandidateNotReferenced,
+    DuplicateContestNames
```

Comment:
You may need to merge in one of these rules and then the other since this line will likely conflict between the two.